device/pcie/pci_device.hpp

```diff
@@ -5,23 +5,23 @@
  */
 
 #pragma once
+
 #include <cstdint>
+#include <cstdio>
 #include <vector>
 
 #include "device/tt_arch_types.h"
 #include "device/architecture_implementation.h"
 
-static uint32_t GS_BAR0_WC_MAPPING_SIZE = (156<<20) + (10<<21) + (18<<24);
-static uint32_t BH_BAR0_WC_MAPPING_SIZE = 188<<21; // Defines the address for WC region. addresses 0 to BH_BAR0_WC_MAPPING_SIZE are in WC, above that are UC
-
-static const uint32_t BH_NOC_NODE_ID_OFFSET = 0x1FD04044;
-static const uint32_t GS_WH_ARC_SCRATCH_6_OFFSET = 0x1FF30078;
-
+// TODO: this is used up in tt_silicon_driver.cpp but that logic ought to be
+// lowered into the PCIDevice class since it is specific to PCIe cards.
```

Comment:
FYI on the redesign.
My idea is that PCIDevice should be rather thin, and the new TTDevice class would hold this.
As your logic was here, so was mine: If it looks fairly low level, it should go out of Chip and Cluster classes (which tt_SiliconDevice ultimately goes towards).
And another logic was, try to put everything arch related (which includes this) into TTDevice abstraction.

No action on this comment.